mypy/test/data.py

```diff
@@ -556,6 +556,9 @@ def repr_failure(self, excinfo: Any) -> str:
 
 
 class DataSuite:
+    def __init__(self, *, update_data: bool = False) -> None:
```

Comment:
I think you should remove those `__init__` definitions that are now a copy of `DataSuite.__init__`, and make the others call `super().__init__(update_data=update_data)`. I think there may be one or two that ignore their `update_data` argument, please figure out if those simply don't need it or if setting it would just be ignored.